src/runtime/graph_executor/graph_executor_factory.cc

```diff
@@ -57,6 +58,14 @@ PackedFunc GraphExecutorFactory::GetFunction(
     return PackedFunc(
         [sptr_to_self, this](TVMArgs args, TVMRetValue* rv) { *rv = this->graph_json_; });
 
+  } else if (name == "get_graph_params") {
```

Comment:
Whats the motivation behind adding this? It does seem useful so I'm all for it but we should add some tests to make sure it works as expected.

Reply:
This function will return params from the compiled module (.so binary). This is the only way to read them from the compiled module. Without this one will not be able to read and set them with `set_params`.
Which tests will you suggest to add?

Reply:
I think it'd be worth adding a test that you can take a compiled module, call `set_params` then `get_graph_params` (which we might want to rename to `get_params` for parity) and confirm the before and after match.